RunForrest/src/js/main.js

```diff
@@ -0,0 +1,30 @@
+function validate(e) {
+    if (e.keyCode===13) callPromise();
```

Comment:
What does 13 mean? Use a const with a significant name for this key code.

Reply:
Don't get used to inline expressions like `if (condition) expression`. Prefer: 
```
if (condition) {
   expression
}
```
Is much safer :)